### DISCUSSION TITLE
Welcome to `cliclack` Discussions!

### DISCUSSION THREAD

**Discussion by fadeevab** [fadeevab/cliclack#35] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cylewitruk** [comment]
Hello!

Not really sure what to write, but my name's Cyle, been developing for about 20 yrs but mostly in the .NET space until recently when I fell in love with Rust :crab: 

I appreciate clean, visually appeasing interfaces, and that's one of the reasons I'm so attracted to this project -- using `inquire` + `indicatif` + `prettytables` + etc. etc. and trying to create nice, uniform styling across these frameworks isn't always the easiest task, so I welcome a nice, clean, opinionated library like `cliclack` which makes this trivial :smile: 

My background is primarily fintech/finance, but nowadays I work on the [Stacks](https://github.com/stacks-network/stacks-core) project, and in-between I try to contribute back to projects that I find valuable.

My goal here is to try to help to evolve `cliclack` by proposing ideas, helping with code reviews, and contributing where/when I can :smile:
